Monika After Story/game/script-topics.rpy

```diff
@@ -1786,12 +1786,20 @@ label monika_rain_holdme:
         m 1dsc "Sorry..."
     return
 
-label monika_holdme_prep:
-    stop music fadeout 1.0
+label monika_holdme_prep(lullaby=True, no_music=True):
 
+    # start the lullaby timer
+    if lullaby and no_music:
+        play music "<silence 1800.0>"
```

Comment:
can you make this and the lullably constants in the songs store (`zz_music_selector`)? then you can access via `store.songs.varname`